src/sentry/models/identity.py

```diff
@@ -36,11 +36,43 @@ class IdentityProvider(Model):
     """
 
     __include_in_export__ = False
+    organization = FlexibleForeignKey("sentry.Organization", blank=True, null=True)
 
+    # To be renamed to "provider" because this is a bad column name
     type = models.CharField(max_length=64)
+    # Replacing "type"
+    provider = models.CharField(max_length=64, null=True)
+
+    # To be renamed to "provider_id", because we're already renaming "type"
+    external_id = models.CharField(max_length=64, null=True)
```

Comment:
Up to you, but you can rename this just in Django by using `provider_id = models.CharField(max_length=64, null=True, db_column='external_id')`

Reply:
We do a lot of debugging/incident response through Redash so I want the column names to match up. IMO I should clean this up while we have the opportunity to do so.

Reply:
I don't really think this needs renamed either, if anything we can throw some comments on top of this stuff, but IMO external_id is fine since this represents the identifier provided to us by the external IDP that this is representing right?